zpeclient.go

```diff
@@ -328,7 +332,7 @@ func (c *Cache) authorize(principal string, check AthenzAccessCheck) (bool, erro
 	for role, members := range domainData.roleToPrincipals {
 		for _, member := range members {
 			if member.memberRegex.MatchString(principal) {
-				if member.expiration.IsZero() || member.expiration.After(time.Now()) {
+				if member.expiration.IsZero() || member.expiration.After(time.Now()) && !member.systemDisabled {
```

Comment:
Shouldn't we put the `member.expiration.IsZero() || member.expiration.After(time.Now())` check into parenthesis? Otherwise, the first `member.expiration.IsZero()` can short circuit if it's true and then the systemDisabled flag will be essentially ignored.

Reply:
+1 on that

Reply:
Oops, my bad. Should be good now - https://github.com/yahoo/k8s-athenz-webhook/pull/24/files#diff-abac4b8aebeabb41b0d0f0baa1d6ec2f2c0920708fa80c5282be51c4c1ff412bR335.